edx_proctoring/static/proctoring/js/exam_action_handler.js

```diff
@@ -0,0 +1,95 @@
+var edx = edx || {};
+
+(function($) {
```

Comment:
Using jquery here seems a little heavy. I'd recommend axios for the ajax call and vanilla js (`getDocumentById` or `querySelector` w/ `.dataset`)